src/libs/actions/Report.ts

```diff
@@ -3714,6 +3715,30 @@ function setGroupDraft(newGroupDraft: Partial<NewGroupChatDraft>) {
     Onyx.merge(ONYXKEYS.NEW_GROUP_CHAT_DRAFT, newGroupDraft);
 }
 
+function buildOptimisticTaskDataForNewAssingee(assigneeLogin: string) {
```

Comment:
Functions that start with `build` should not set onyx data. Either move the onyx setting process to where this function is called or change its name to something more fitting

Reply:
Updated the name of the function.